src/rendition-mixin.js

```diff
@@ -66,6 +67,10 @@ class Representation {
 
     this.bandwidth = playlist.attributes.BANDWIDTH;
 
+    this.codecs = codecsForPlaylist(mpc.master(), playlist);
+
+    this.playlist = playlist;
```

Comment:
Do we want to add the whole playlist object to the representation?

Reply:
It makes it easier to pass to `smoothQualityChange`/`fastQualityChange`